compiler/optimizer/OMROptimizer.cpp

```diff
@@ -2050,6 +2050,10 @@ int32_t OMR::Optimizer::performOptimization(const OptimizationStrategy *optimiza
          }
 
       delete opt;
+      // we cannot easily invalidate during IL gen since we could be peeking and we cannot destory our
+      // caller's alias sets
+      if (!isIlGenOpt())
```

Comment:
How does the case when we are doing IL gen for inlining get handled ? i.e. could we be destroying our caller's alias sets (as the comment suggests) in that case ?